docs/modules/ROOT/pages/extensions/activemq.adoc

```diff
@@ -0,0 +1,27 @@
+// Do not edit directly!
+// This file was generated by camel-quarkus-package-maven-plugin:update-extension-doc-page
+
+[[activemq]]
+= ActiveMQ
+
+*Since Camel Quarkus 1.0.0-M5, JVM and native*
+
+Send messages to (or consume from) Apache ActiveMQ. This component extends the Camel JMS component.
+
+== What's inside
```

Comment:
Maybe 'Included components'? I don't want to get too hung up on wording so if you disagree, that's ok.

Reply:
The problem with "components" on this level is that technically not only components can be listed here: there can also be languages, data formats and "others". How about `Included Camel bits`?

Reply:
Yes, true.... I'm happy to leave it as-is for now.